pinax/stripe/tests/test_actions.py

```diff
@@ -1666,17 +1666,17 @@ def test_sync_customer_purged_remotely_not_locally(self, RetrieveMock, SyncPayme
         self.assertFalse(SyncSubscriptionMock.called)
         self.assertTrue(PurgeLocalMock.called)
 
-    @patch("pinax.stripe.actions.invoices.sync_invoice_from_stripe_data")
+    @patch("stripe.Invoice.auto_paging_iter")
     @patch("stripe.Customer.retrieve")
-    def test_sync_invoices_for_customer(self, RetreiveMock, SyncMock):
-        RetreiveMock().invoices().data = [Mock()]
+    def test_sync_invoices_for_customer(self, RetrieveMock, SyncMock):
+        RetrieveMock.return_value = [Mock()]
```

Comment:
I'm not sure this is correct. I believe that `stripe.Customer.retrieve` isn't actually called anymore. You would need to keep the `@patch("pinax.stripe.actions.invoices.sync_invoice_from_stripe_data")` and remove the `@patch("stripe.Customer.retrieve")` patch, which would mean that `stripe.Invoice.auto_paging_iter` is set to the `RetrieveMock` and `pinax.stripe.actions.invoices.sync_invoice_from_stripe_data` is set to the `SyncMock`.